addins/Cake.Issues.DocFx.yml

```diff
@@ -3,9 +3,20 @@ NuGet: Cake.Issues.DocFx
 Assemblies:
 - "/**/Cake.Issues.DocFx.dll"
 Repository: https://github.com/cake-contrib/Cake.Issues.DocFx
-Author: BBT Software AG
-Description: "The DocFx support for the Cake Issues Addin allows you to read warnings logged by DocFx."
+Author: BBT Software AG and contributors
+Description: "The MsBuild support for the Cake.Issues addin for Cake allows you to read issues logged as warnings in a MsBuild log.
+
+This addin provides the aliases for reading warnings from DocFx log files and provides them to the Cake.Issues addin.
+It also requires the core Cake.Issues addin.
+
+There are also additional addins for generating reports or posting issues to pull requests.
+
+See the Project Site for an overview of the whole ecosystem of addins for working with issues in Cake scripts."
 Categories:
-- Issues
-- Documentation
+- Cake
+- Script
+- Cake-Issues

```

Comment:
How to proceed which these kind of tags? A tag `Cake-Issues` or `Cake-IssueProvider` serves a purpose in the nuget.org ecosystem to have these addins grouped together, while in the context of the Cake website the `Cake` prefix is only confusing as a category.

Reply:
I can filter out these tags

Reply:
Don't think filtering out is the best solution here. As a category `Issues` makes sense on the Cake website (like it has now). But I also don't have a silver bullet how to proper map NuGet tags to website categories

Reply:
How about I strip the `Cake-` prefix?

Reply:
@Jericho That would be perfect in this case